examples/cheat-sheet/maven-security-cheat-sheet.adoc

```diff
@@ -1,5 +1,7 @@
 = Cheat Sheet: 10 Maven Security Best Practices
-Simon Maple <https://twitter.com/sjmaple> @sjmaple, Java Champion and Developer Advocate at Snyk;Robert Scholte <https://twitter.com/rfscholte> @rfscholte, CEO of Sourcegrounds, Chairman of the Apache Maven project
+Simon Maple <https://twitter.com/sjmaple>; Robert Scholte <https://twitter.com/rfscholte>
```

Comment:
@mojavelinux Hopefully we should be able to write the following:

```
Simon Maple, Java Champion and Developer Advocate at Snyk <https://twitter.com/sjmaple>; Robert Scholte, CEO of Sourcegrounds, Chairman of the Apache Maven project <https://twitter.com/rfscholte>
```

But using a dedicated attribute is not so bad. It's avoid headache if the author's name contains comma :grimacing:

Reply:
Yeah, that's kind of stretching the shorthand to the point that it doesn't become all that useful anyway. So it's better just to use separate attributes. (Though, even then, some improvements in AsciiDoc would be helpful).